mep/footnotes/templates/footnotes/card_list.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'base.html' %}
+{% load static humanize widget_tweaks mep_tags %}
+{% load render_bundle from webpack_loader %}
+
+{% block js %}
+{% render_bundle 'cardSearch' 'js' %}
+{% endblock %}
+
+{% block page-subtitle %}Cards · {% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title="cards" style="search" %}
+{% endblock %}
+
+{% block main-style %}white{% endblock %}
+
+{% block content %}
+{% include 'snippets/breadcrumbs.html' %}
+
+<section class="cards search-form">
+    <form id="cards-form">
+        {# keyword search disabled for now #}
+        {# {% render_field form.query %} #}
+        <label for="{{ form.query.id_for_label }}"></label>
+        {% include 'common/form_errors.html' %}
+        <fieldset class="group">
+        {% comment %}
```

Comment:
do we need all of this here, commented out? can always just copy it over from members or books when we're ready.

Reply:
We don't need it, but won't be adding it soon?

Reply:
ah, I guess I didn't notice that the code here actually will fit this form perfectly. fine to leave it in then.